kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateList.js

```diff
@@ -67,59 +75,103 @@ define([
         );
     }
 
+    function createTableRow(tableBody, id) {
```

Comment:
moved up from further down in the file, and added in the `tableBody` variable to make the function more self-contained